deepcell_label_utils/utils_test.py

```diff
@@ -0,0 +1,222 @@
+from utils import SpatialLabelConverter, mask_to_polygons, polygons_to_mask
+from shapely.geometry import Point
+
+import numpy as np
+import pandas as pd
```

Comment:
General rules for organizing imports
- Three groups: 1) Native python imports, 2) External packages, 3) Imports within this package
- Sort alphabetically within groups

```suggestion
import numpy as np
import pandas as pd
from shapely.geometry import Point

from utils import SpatialLabelConverter, mask_to_polygons, polygons_to_mask
```